packages/capabilities/src/http.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * HTTP Capabilities.
+ *
+ * These can be imported directly with:
+ * ```js
+ * import * as HTTP from '@web3-storage/capabilities/http'
+ * ```
+ *
+ * @module
+ */
+import { capability, Schema, ok } from '@ucanto/validator'
+import { content } from './blob.js'
+import { equal, equalBody, equalWith, SpaceDID, Await, and } from './utils.js'
+
+/**
+ * `http/put` capability invocation MAY be performed by any agent on behalf of the subject.
+ * The `blob/add` provider MUST add `/http/put` effect and capture private key of the
+ * `subject` in the `meta` field so that any agent could perform it.
```

Comment:
I suggest we remove reference to `blob/add` here as it introduces additional context which I'm not sure helps here.

```suggestion
 * `http/put` capability invocation MAY be performed by any authorized agent on behalf of the subject
 * as long as they have referenced `body` content to do so.
```